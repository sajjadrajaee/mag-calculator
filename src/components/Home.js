import React from 'react';

function Home() {
  return (
    <div className="about-container">
      <h3>Welcome to Magiciant Calculator!</h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nec mauris bibendum mollis.
        Vivamus quis fermentum urna, at scelerisque nunc. Sed sit amet ante nisi.
        Mauris a sapien vitae metus vulputate efficitur.
        Donec viverra erat sed porta ultricies. Donec at turpis velit. Nulla et vulputate nisi.
        Vestibulum in efficitur leo, sit amet placerat nibh. Fusce in ligula massa.
        Vestibulum nulla orci, pretium a velit in, egestas sollicitudin odio.
        Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
      </p>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nec mauris bibendum mollis.
        Vivamus quis fermentum urna, at scelerisque nunc. Sed sit amet ante nisi.
        Mauris a sapien vitae metus vulputate efficitur.
        Donec viverra erat sed porta ultricies. Donec at turpis velit. Nulla et vulputate nisi.
        Vestibulum in efficitur leo, sit amet placerat nibh. Fusce in ligula massa.
        Vestibulum nulla orci, pretium a velit in, egestas sollicitudin odio.
        Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
        {' '}

      </p>
    </div>
  );
}

export default Home;
