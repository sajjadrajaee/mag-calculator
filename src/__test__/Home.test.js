import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Home from '../components/Home';

it('render to match the Home', () => {
  const home = render(
    <BrowserRouter>
      <Home />
    </BrowserRouter>,
  );
  expect(home).toMatchSnapshot();
});
