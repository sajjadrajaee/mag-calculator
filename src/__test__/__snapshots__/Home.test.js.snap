// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render to match the Home 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="about-container"
      >
        <h3>
          Welcome to Magiciant Calculator!
        </h3>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nec mauris bibendum mollis. Vivamus quis fermentum urna, at scelerisque nunc. Sed sit amet ante nisi. Mauris a sapien vitae metus vulputate efficitur. Donec viverra erat sed porta ultricies. Donec at turpis velit. Nulla et vulputate nisi. Vestibulum in efficitur leo, sit amet placerat nibh. Fusce in ligula massa. Vestibulum nulla orci, pretium a velit in, egestas sollicitudin odio. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
        </p>
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nec mauris bibendum mollis. Vivamus quis fermentum urna, at scelerisque nunc. Sed sit amet ante nisi. Mauris a sapien vitae metus vulputate efficitur. Donec viverra erat sed porta ultricies. Donec at turpis velit. Nulla et vulputate nisi. Vestibulum in efficitur leo, sit amet placerat nibh. Fusce in ligula massa. Vestibulum nulla orci, pretium a velit in, egestas sollicitudin odio. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
           
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="about-container"
    >
      <h3>
        Welcome to Magiciant Calculator!
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nec mauris bibendum mollis. Vivamus quis fermentum urna, at scelerisque nunc. Sed sit amet ante nisi. Mauris a sapien vitae metus vulputate efficitur. Donec viverra erat sed porta ultricies. Donec at turpis velit. Nulla et vulputate nisi. Vestibulum in efficitur leo, sit amet placerat nibh. Fusce in ligula massa. Vestibulum nulla orci, pretium a velit in, egestas sollicitudin odio. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
      </p>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nec mauris bibendum mollis. Vivamus quis fermentum urna, at scelerisque nunc. Sed sit amet ante nisi. Mauris a sapien vitae metus vulputate efficitur. Donec viverra erat sed porta ultricies. Donec at turpis velit. Nulla et vulputate nisi. Vestibulum in efficitur leo, sit amet placerat nibh. Fusce in ligula massa. Vestibulum nulla orci, pretium a velit in, egestas sollicitudin odio. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
         
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
